pages/clubs/Running.jsx

```diff
@@ -9,39 +11,28 @@ var Running = React.createClass({
   contextTypes: {
     intl: React.PropTypes.object
   },
-  setData : function(data){
-    for(var i = 0; i < data.length; i++){
-      var categoryLabel = data[i].category.toLowerCase();
+  componentDidMount: function() {
+    this.props.teachAPI.getClubsGuides(function(err, res){
+      if(err) {
+        this.setState({error: err});
+        return;
+      }
 
-      categoryLabel = "guide_category_" + categoryLabel.split(' ').join('_');
+      var data = res.body;
```

Comment:
even if the status code is 200, that does not mean the body has anything in it. You can have an empty document response despite HTTP200